dev/redis-trib.rb

```diff
@@ -0,0 +1,1388 @@
+#!/usr/bin/env ruby
```

Comment:
Is this a copy of redis-trib.rb from https://github.com/antirez/redis/blob/unstable/src/redis-trib.rb ?